Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "GeoInterface"
 uuid = "cf35fbd7-0cd7-5166-be24-54bfbe79505f"
 authors = ["JuliaGeo and contributors"]
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
Best not to bump versions in PRs because we always forget and bump again later, e.g. for the last ArchGDAL version.